backend/api/user.go

```diff
@@ -15,13 +15,13 @@ import (
 // @Tags         user
 // @Accept       json
 // @Produce      json
-// @Param   	 username 	  formData string true "user's name"
-// @Param   	 password 	  formData string true "password"
-// @Param   	 email 	  	  formData string true "email"
-// @Param   	 overview 	  formData string false "overview"
-// @Param   	 organization formData string false "organization"
+// @Param   	 username 	  	formData string true "user's name"
+// @Param   	 password 	  	formData string true "password"
+// @Param   	 email 	  	  	formData string true "email"
+// @Param   	 overview 	  	formData string false "overview"
+// @Param   	 institutionID 	formData string false "institution ID"
```

Comment:
I think it is possible for informing the default value.
https://github.com/swaggo/swag#attribute

Reply:
As mentioned above.